.ibm/pipelines/windows-test.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash 
+set -x
+apt-get update -y
+apt-get install -y sshpass
+
+LOGFILE="pr-${GIT_PR_NUMBER}-windows-tests-${BUILD_NUMBER}"
+export REPO=${REPO:-"https://github.com/anandrkskd/odo"}
+#copy test script inside /tmp/
+#sshpass -p $PASS scp -o StrictHostKeyChecking=no ./.ibm/pipelines/openshift-windows-tests.sh    Administrator@161.156.170.160:/tmp/openshift-windows-tests-${BUILD_NUMBER}.sh
+sshpass -p $PASS scp -o StrictHostKeyChecking=no ./.ibm/pipelines/windows-test-script.ps1    Administrator@161.156.170.160:/tmp/windows-test-script.ps1
```

Comment:
PASS variable name is not clear. Please use a more verbose one, something like `WINDOWS_PASSWORD`

Reply:
Understood! 
Working on it.